auto-deploy-docker-ngnix/README.md

```diff
@@ -0,0 +1,40 @@
+# Auto deployment of an Ubuntu VM with Ngnix Docker container 
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fauto-deploy-docker-ngnix%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fauto-deploy-docker-ngnix%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template deploy an Ubuntu VM with Docker (using the Docker, and  custom-script Extension). The script will download HTML file(s), configure, and run Ngnix container.
+
+![Diagram](/auto-deploy-docker-ngnix/images/Docker-in-Azure2.png "Complete diagram with external resources")
+
+### Prerequisites
+
+- Create a secure Azure acount Storage and have the access key.
+- Edit the `mynginx.sh` script and change EXISTING_STORAGE_ACCOUNTNAME and  EXISTING_SCRIPT_STORAGE_ACCOUNT_KEY.
```

Comment:
It's better if you modify this template to download the script off of this repo, so that the user does not need to do pre-setup and this template can be run using the "Deploy to Azure" button.